### PR TITLE
Update compilers

### PR DESCRIPTION
**Pull request by VoR0220**
* solves https://github.com/eris-ltd/eris-cli/issues/1100
* solves https://github.com/eris-ltd/eris-cli/issues/997 and thus lays the base to solve https://github.com/eris-ltd/eris-cli/issues/1054

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1,46 +1,89 @@
...
+  version: 8ca15f5455104403db4202c995e2f6e161654c02
+  subpackages:
+  - go/docs
+- name: github.com/eris-ltd/eris-cli
+  version: 002c63e2d0850b33c279c29fee52dd30853e100d
   subpackages:
-  - go/common
+  - config
+  - version
 - name: github.com/eris-ltd/eris-logger
```
Comment (zramsay) [inline glide.lock]: need to deprecate `eris-logger`

Comment (VoR0220) [inline glide.lock]: I do...started it. Is the CLI logger the same as eris-logger?

File: perform/cache.go [hunk 2]
```diff
@@ -1,18 +1,17 @@
-package util
+package perform
 
 import (
 	"io/ioutil"
 	"os"
 	"path"
-	"strings"
-	//log "github.com/eris-ltd/eris-logger"
+	def "github.com/eris-ltd/eris-compilers/definitions"
```
Comment (zramsay) [inline perform/cache.go]: we stopped using shorthands for easier code readability.

File: perform/client.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package perform
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	def "github.com/eris-ltd/eris-compilers/definitions"
```
Comment (zramsay) [inline perform/client.go]: drop shorthand

File: perform/server.go [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 	"encoding/json"
 	"io/ioutil"
 	"net/http"
 	"os"
 
-	"github.com/eris-ltd/eris-compilers/util"
+	def "github.com/eris-ltd/eris-compilers/definitions"
 
-	"github.com/eris-ltd/common/go/common"
+	"github.com/eris-ltd/eris-cli/config"
```
Comment (zramsay) [inline perform/server.go]: ++ on getting rid of common. not sure we want to be importing cli. if we do, then `eris-logger` can be `github.com/eris-ltd/eris-cli/log`

Comment (VoR0220) [inline perform/server.go]: Should we just be using plain fmt.Println if we don't want to be importing these things?

**Comment by zramsay** [comment]
mostly LGTM
needs:
- [ ] split the PR into path re-write and dockerfile PR
- [ ] `def => definitions` everywhere
- [ ] unsure what we do about logs/importing cli

**Pull request closed without merging**